bin/git-overwritten

```diff
@@ -22,6 +22,9 @@ unset head
 unset base
 unset diff_type
 
+# Figure out what argument to use for date as GNU & BSD `date`s are different
+date --version 2>&1 >/dev/null && DATEARG='-d@' || DATEARG='-r'
```

Comment:
What if BSD `date` adds `--version` in the future?

How about this feature check:

```sh
[ "$(date -r123 "+%s" 2>/dev/null)" -eq "123" ] && DATEARG='-r'
```